src/datadog-step-functions.ts

```diff
@@ -155,6 +158,28 @@ export class DatadogStepFunctions extends Construct {
       )
     );
   }
+
+  private setTags(stateMachine: sfn.StateMachine): void {
+    const props = this.props;
+    if (props.env) {
+      Tags.of(stateMachine).add(DD_ENV_ENV_VAR, props.env);
+    }
+    if (props.service) {
+      Tags.of(stateMachine).add(DD_SERVICE_ENV_VAR, props.service);
+    }
+    if (props.version) {
+      Tags.of(stateMachine).add(DD_VERSION_ENV_VAR, props.version);
```

Comment:
My understanding is that these three need to be added to the function as env vars, not as part of the dd tags. But I also don't have a good enough understanding of our step function instrumentation. Can you help me understand the decision to place all the tags in the location you did?